draft-yasskin-http-origin-signed-responses.md

```diff
@@ -893,51 +942,20 @@ error (Section 5.4.2 of {{!RFC7540}}) of type PROTOCOL_ERROR, as described in
 Section 8.2 of {{?RFC7540}}.
 
 Instead, the client MUST validate such a PUSH_PROMISE and its response by
-parsing the `Signature` header into a list of signatures according to the
-instructions in {{signature-validity}}, and searching that list for a valid
-signature using the algorithm in {{authority-chain-validation}}. If no valid
-signature is found, the client MUST treat the response as a stream error
+passing the exchange they represent to the algorithm in {{cross-origin-trust}}.
```

Comment:
@nyaxt pointed out that this needs to exclude the `Signature` header from the exchange passed to {{cross-origin-trust}} (since you can't cover a signature with itself), so I'm working on that.

Reply:
Ok, fixed, I think.